src/components/Main.js

```diff
@@ -29,32 +34,70 @@ export default function Main() {
   const [coinList, setCoinList] = useState([]);
   const [searchCoin, setSearchCoin] = useState("");
 
+  const dispatch = useDispatch();
+
+  const tickerCoinList = useSelector((state) => state.socket.coinList);
+  const realTimeCoin = useSelector((state) => state.socket.socketCoin);
+
+  useEffect(() => {
+    const myCoin = "ALL";
+
+    const fetchCoinList = function () {
+      dispatch(requestCoinList(myCoin));
+    };
+
+    fetchCoinList();
+  }, []);
+
+  useEffect(() => {
+    const tickerCoin = JSON.stringify(tickerCoinList);
+    const parseTickerCoin = JSON.parse(tickerCoin);
```

Comment:
```suggestion
    const tickerCoin = JSON.parse(JSON.stringify(tickerCoinList));
```
이것도 한 줄로 줄일 수 있을 거 같아요 !
하지만 웅민님이 명시적으로 작성해주시기 위해서 저렇게 작성해주신 거 같은데
그런 생각으로 작성해주셨다면 그냥 놔둬도 괜찮을 거 같습니다 !